variables.tf

```diff
@@ -156,6 +156,46 @@ variable "autoscaling_indexes" {
   default     = {}
 }
 
+variable "schedule_scaling_read" {
```

Comment:
Please change the variable names, type and default to match existing variables like `autoscaling_indexes`

Reply:
```suggestion
variable "autoscaling_scheduled_read" {
```

Reply:
Adjusted